pkgs/exercism/cli/registry.yaml

```diff
@@ -0,0 +1,19 @@
+packages:
+  - type: github_release
+    repo_owner: exercism
+    repo_name: cli
+    description: A Go based command line tool for exercism.org
+    asset: exercism-{{trimV .Version}}-{{.OS}}-{{.Arch}}.{{.Format}}
+    format: tar.gz
+    overrides:
+      - goos: windows
+        format: zip
+    replacements:
+      amd64: x86_64
+      386: i386
+    checksum:
+      type: github_release
+      asset: exercism_checksums.txt
+      algorithm: sha256
+    files:
+      - name: exercism
```

Comment:
Please add a newline at the end of files.

<img width="295" alt="image" src="https://github.com/aquaproj/aqua-registry/assets/13323303/1729c5b5-db2b-4299-a34b-1ab853c91884">

Reply:
I think vscode is removing or prettier is removing it. 
Suggest an .editorconfig that defines this and should honor it. I'll try it out and if it fixes it for me I'll contribute it. thanks!